### PR TITLE
@kanaabe => Gucci Layout

### PR DESCRIPTION
**Pull request by eessex**
Sets up layouts and mobile styles. Adds components for...

Partner: 
- Header (artsy logo / partner logo)
- Footer (presented by / partner logo )

Series:
- Header (main header w logos/series title)
- Footer (about)
- Nav (section titles)

Section:
- Container
- Video (and video preview)
- Text (featuring/share/about)

TODO: 
- Nothing here is tested.
- Header table of contents/jump links
- Reevaluate breakpoints in relation to grid

gucci

### CODE REVIEW

File: desktop/apps/editorial_features/components/gucci/components/partner/partner_header.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    img {
+      max-width: 100px;
+      margin-left 20px;
+    }
+    .divider:after {
+      content: '+';
+      margin-left: 20px;
+    }
+  }
+`
```
Comment (kanaabe) [inline desktop/apps/editorial_features/components/gucci/components/partner/partner_header.jsx]: Definitely excited to see these transfer to Reaction

File: desktop/apps/editorial_features/components/gucci/components/section/section_text.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <Row className='SectionText'>
+        <Col sm={4} className='col col--first'>
+          <div>
+            <Title>Featuring</Title>
+            <Featuring>{section.featuring}</Featuring>
+          </div>
+          <ShareContainer>
+            <Title>Share</Title>
+            <Share
+              url='http://artsy.net/gender-equality'
```
Comment (kanaabe) [inline desktop/apps/editorial_features/components/gucci/components/section/section_text.jsx]: `${APP_URL}/gender-equality` would be more reliable here

Comment (kanaabe) [reply desktop/apps/editorial_features/components/gucci/components/section/section_text.jsx]: Also, this should include the slug, right?

File: desktop/apps/editorial_features/components/gucci/components/series/series_footer.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            logo={logoSrc}
+            url={curation.partner_link_url}
+          />
+        </Col>
+        <Col sm={7} className='col col--last'>
+          <Text html={curation.about} />
+          <PartnerFooter
+            logo={logoSrc}
+            url={curation.partner_link_url}
+          />
```
Comment (kanaabe) [inline desktop/apps/editorial_features/components/gucci/components/series/series_footer.jsx]: Are there supposed to be two PartnerFooters?

Comment (eessex) [reply desktop/apps/editorial_features/components/gucci/components/series/series_footer.jsx]: Yeah it is to deal with position changes in desktop v mobile -- ideally our container would know which one it is, but maybe I can get that into a later PR

File: desktop/apps/editorial_features/components/gucci/components/series/series_header.jsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  align-items: center;
+  .PartnerHeader {
+    flex: 1;
+  }
+  .title {
+    flex: 2;
+    text-align: center;
+    font-size: 23px;
+    text-transform: uppercase;
+    height: min-content;
```
Comment (kanaabe) [inline desktop/apps/editorial_features/components/gucci/components/series/series_header.jsx]: Is this more of a nice to have? The support for `min-content` seems a little weak https://caniuse.com/#search=min-content

**Pull request merged**